src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/SpatialCoordinateSystemManager.cs

```diff
@@ -1,394 +1,396 @@
+        {
+            DebugLog("LocalizeAsync");
+            if (currentLocalizationSession != null)
+            {
+                Debug.LogError($"Failed to start localization session because an existing localization session is in progress");
+                return Task.FromResult(false);
+            }
+
+            if (!participants.TryGetValue(socketEndpoint, out SpatialCoordinateSystemParticipant participant))
+            {
+                Debug.LogError($"Could not find a SpatialCoordinateSystemParticipant for SocketEndpoint {socketEndpoint.Address}");
+                return Task.FromResult(false);
+            }
+
+            if (!localizers.TryGetValue(spatialLocalizerID, out ISpatialLocalizer localizer))
+            {
+                Debug.LogError($"Could not find a ISpatialLocalizer for spatialLocalizerID {spatialLocalizerID}");
+                return Task.FromResult(false);
+            }
+
+            DebugLog("Returning a localization session.");
+            return RunLocalizationSessionAsync(localizer, settings, participant);
+        }
+
+        protected override void OnDestroy()
+        {
+            foreach (INetworkManager networkManager in networkManagers)
+            {
+                UnregisterEvents(networkManager);
+            }
+
+            CleanUpParticipants();
+        }
+
+        private void Update()
+        {
+            foreach (var participant in participants.Values)
+            {
+                participant.EnsureStateChangesAreBroadcast();
+            }
+        }
+
+        private void SendLocalizationCompleteCommand(SocketEndpoint socketEndpoint, bool localizationSuccessful)
+        {
+            using (MemoryStream stream = new MemoryStream())
+            using (BinaryWriter message = new BinaryWriter(stream))
+            {
+                message.Write(LocalizationCompleteCommand);
+                message.Write(localizationSuccessful);
+
+                socketEndpoint.Send(stream.ToArray());
+            }
+        }
+
+        private void OnConnected(SocketEndpoint endpoint)
+        {
+            if (participants.ContainsKey(endpoint))
+            {
+                Debug.LogWarning("SpatialCoordinateSystemParticipant connected that already existed");
+                return;
+            }
+
+            DebugLog($"Creating new SpatialCoordinateSystemParticipant, IPAddress: {endpoint.Address}, DebugLogging: {debugLogging}");
+
+            SpatialCoordinateSystemParticipant participant = new SpatialCoordinateSystemParticipant(endpoint, debugVisual, debugVisualScale);
+            participants[endpoint] = participant;
+            participant.ShowDebugVisuals = showDebugVisuals;
+            participant.SendSupportedLocalizersMessage(endpoint, localizers.Keys);
+
+            participant.SendSupportedLocalizersMessage(endpoint, localizers.Keys);
+

```

Comment:
Looks like this got duplicated as part of a merge, the call should only happen once